Tests/Swift Tests/BugsnagSwiftPublicAPITests.swift

```diff
@@ -257,7 +257,7 @@ class BugsnagSwiftPublicAPITests: XCTestCase {
         client.notifyError(err) { (event) -> Bool in return false }
      
         client.leaveBreadcrumb(withMessage: "msg")
-        client.leaveBreadcrumb("msg", metadata: [:], type: .manual)
+        client.leaveBreadcrumb(withMessage: "msg", metadata: [:], andType: .manual)
```

Comment:
Why has this changed; have we made a breaking change to the API?

Reply:
Xcode refused to compile if I didn't change this.

`Incorrect argument labels in call (have '_:metadata:type:', expected 'withMessage:metadata:andType:')`

Reply:
Since the test passes on master, some change in this branch broke the public API, probably around making the sink public without `NS_SWIFT_NAME`, as we have here: https://github.com/bugsnag/bugsnag-cocoa/blob/b83847c74669f64a9afc0e7e7ade595b620c09e8/Bugsnag/include/Bugsnag/BugsnagClient.h#L115-L118